### DISCUSSION TITLE
E-mail Subject in Password Request E-Mail

### DISCUSSION THREAD

**Discussion by AndreyKindin** [directus/directus#7344] [Feature Requests]
Hello,
it would be great if it would be possible to change the subject in the password request email.

```
/api/src/controllers/auth.ts
178 - await service.requestPasswordReset(req.body.email, req.body.reset_url || null);
178 + await service.requestPasswordReset(req.body.email, req.body.reset_url || null, req.body.subject || null);
```

```
/packages/sdk/src/handlers/passwords.ts
10 - async request(email: string, reset_url?: string | null): Promise<void> {
10 + async request(email: string, reset_url?: string | null, subject?: string | null): Promise<void> {

11 - await this.transport.post('/auth/password/request', { email, reset_url });
11 + await this.transport.post('/auth/password/request', { email, reset_url, subject });
```

**Comment by JaimeBulaGooten** [comment]
@AndreyKindin Did you manage to solve this?

**Reply by JaimeBulaGooten** [reply]
I just went in into the sources, this is hard coded. I did saw that the implementation is there, it is just that it hasn't been exposed.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
